packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -45,6 +46,7 @@ export const zoomMeetingsSchema = z.object({
 });
 
 // Successful API response
+// @TODO: add link to the docs
```

Comment:
This both TODO can wait a little, updating them in the next iteration of this bugfix.